src/Core/src/Platform/Windows/KeyboardAcceleratorExtensions.cs

```diff
@@ -0,0 +1,132 @@
+﻿using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Windows.System;
+
+namespace Microsoft.Maui.Platform
+{
+	public static class KeyboardAcceleratorExtensions
+	{
+		public static void UpdateAccelerator(this MenuFlyoutItemBase platformView, IMenuFlyoutItem menuFlyoutItem)
+		{
+			var keyboardAccelerators = menuFlyoutItem.Accelerators?[0].ToPlatform();
+
+			if (keyboardAccelerators is null)
+				return;
+
+			foreach (var keyboardAccelerator in keyboardAccelerators)
+				platformView.KeyboardAccelerators.Add(keyboardAccelerator);
+		}
+
+		// Single key (A, Delete, F2, Spacebar, Esc, Multimedia Key) accelerators and multi-key
+		// accelerators (Ctrl+Shift+M) are supported.
+		// Gamepad virtual keys are not supported.
+		public static IList<KeyboardAccelerator>? ToPlatform(this IAccelerator accelerator)
```

Comment:
I see the code paths always add 1 item, so we can also simply this and make it :

```suggestion
		public static KeyboardAccelerator? ToPlatform(this IAccelerator accelerator)
```

Reply:
Makes sense to me, unless we do make Windows support multiple accelerators. @jsuarezruiz were you intending on building Windows support for multiple accelerators either now or in the future?

Reply:
Just added support to multiple accelerators based on a previous comment. Any preference?, feedback again it?

Reply:
could you add a sample in XAML and in c# so we can see the usage?

Reply:
The ToPlatform method takes a single accelerator and then also returns a single accelerator? There is a lst being created internally, but it does only ever create 1. If there are multiple accelerators then there will also be multiple IAccelerators. 

Or maybe I am confusing it all, I think there is a 1:1 mapping of IAccelerator and the native side?

But it all depends on the usage that @jsuarezruiz shows us :)